app/modules/feed-items/components/FeedItem.js

```diff
@@ -48,18 +60,21 @@ const PureFeedItem = (props: Props): React.Node => {
     default: predicate = 'acted on';
   }
 
+  const imageHash = md5(_.trim(userNameEmail.email).toLowerCase());
+
   return (
     <Feed.Event>
       <Feed.Label>
-        <img src={professor2} alt="profile" />
+        <img src={`https://www.gravatar.com/avatar/${imageHash}`} alt="profile" />
```

Comment:
Small remark: I suppose Gravatar lets you request an image of a particular size?